app/HutchManager/Services/RquestPollingService.cs

```diff
@@ -135,8 +142,9 @@ private void StartTimer()
         private void StopTimer()
           => _timer?.Change(Timeout.Infinite, 0);
         
-        public void SendToQueue(RquestQueryTask jobPayload,string queueName)
+        public async void SendToQueue(RquestQueryTask jobPayload,string queueName)
```

Comment:
@vpnu @beforan does this have to return `Task` or is `void` okay here? https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/async-return-types

Reply:
great question, should be `Task`.

basically always use `async Task` unless you are absolutely certain `async void` is ok, and have checked with an expert 😆 

Elsewhere in this same class we **do** use `async void` and you'll notice I left a comment explaining why, and that it was ok, with a link. `async void` can really mess you around if used in the wrong place.